src/library/mixin_helper.lua

```diff
@@ -561,6 +562,24 @@ function mixin_helper.to_fcstring(value, fcstr)
     return fcstr
 end
 
+--[[
+% to_string
+
+Casts a value to a Lua string. If the value is an `FCString`, it returns `LuaString`, otherwise it calls `tostring`.
+
+@ value (any)
+@ [fcstr] (FCString) An optional `FCString` object to populate to skip creating a new object.
+: (FCString)
+]]
+
+function mixin_helper.to_string(value)
+    if mixin_helper.is_instance_of(value, "FCString") then
+        return value.LuaString
+    end
+
+    return tostring(value)
```

Comment:
Huh, I thought I wrote this function once upon a time. I must not have submitted it. Anyway, a `nil` value should be cast to an empty string, so this line would need to be changed to:
```lua
return value == nil and "" or tostring(value)
```